solution_runner/commands/setup_command.py

```diff
@@ -0,0 +1,130 @@
+from datetime import datetime
+from pathlib import Path
+
+import click
+import requests
+
+from . import consts
+from .consts import Directories, FileExtensions
+from .defaults_and_choices import get_default_year
+from .. import utils
+
+
+_default_year = get_default_year()
+
+
+@click.command(name='setup')
+@click.option('-y', '--year', type=click.IntRange(consts.FIRST_AOC_YEAR, _default_year), default=_default_year,
+              show_default=f'last year: {_default_year}', help='year of puzzle setting up solution for')
+@click.option('-d', '--day', type=click.IntRange(1, 25), required=True, help='day of puzzle setting up solution for')
```

Comment:
what is 1, 25? consider storing them as consts

Reply:
Thanks, fixed in a59edd00.